### PR TITLE
Updates d3-sankey-circular to 0.33.0

### LINKED ISSUE #3610: D3-sankey-circular 0.32.0 breaks webpack builds due to mjs imports
The previous release of d3-sankey-circular (0.32.0) breaks webpack builds when it tries to import named exports from non EcmaScript module #https://github.com/tomshanley/d3-sankey-circular/issues/32 .

This was resolved in #https://github.com/tomshanley/d3-sankey-circular/pull/33 and has been released under version 0.33.0 https://github.com/tomshanley/d3-sankey-circular/commit/dbb238a97da7177b6b69dd00cb81106b1253f97e

Resolving d3-sankey-circular to 0.33.0 fixes the issue for me.

### PR DESCRIPTION
**Pull request by RussGlover**
Fixes #3610

### CODE REVIEW

**Comment by etpinard** [comment]
Thanks very much for the PR!

That `package-lock.json` shows a nasty diff. Which `npm` version are you using? Could you try to generate the `package-lock.json` again using `npm@6.8.0`? Thank you!

**Comment by RussGlover** [comment]
Oops I use yarn usually, didn't think to check about npm version. I'll do that.

**Comment by RussGlover** [comment]
Okay I fixed it, only 7 line diff for package-lock.json now! I went a pretty roundabout way to revert the old commit and had to force push it. Let me know if there is anything else I need to do. Thanks for the quick response!

File: package-lock.json [hunk 1]
```diff
@@ -9311,6 +9311,7 @@
     "regl-error2d": {
       "version": "2.0.6",
       "resolved": "git://github.com/gl-vis/regl-error2d.git#0d27313d932fdb8ffaed17931b7258958bce0b88",
+      "integrity": "sha1-2ydl+hP2aW6JTHSM16fEWhq9ouw=",
```
Comment (antoinerg) [inline package-lock.json]: @RussGlover can you run `npm i` again? This line shouldn't be here!

Comment (RussGlover) [reply package-lock.json]: I tried again but that line keeps getting changed

Comment (antoinerg) [reply package-lock.json]: What version of `npm` are you using?

Comment (RussGlover) [reply package-lock.json]: Its happening with 6.8.0 and 6.9.0. I was able to update it under npm in wsl and the integrity diff went away.

Comment (antoinerg) [reply package-lock.json]: I couldn't reproduce this behavior with npm 6.8.0 but I'm running Linux and it seems like you are on Windows :smile: Thanks again @RussGlover for reporting this issue!

Comment (RussGlover) [reply package-lock.json]: Okay everything should be good. I wasn't sure how to revert the bad commits and squash them into one commit without force pushing.

**Comment by antoinerg** [comment]
@etpinard I checked the changes upstream and this is good to go. I will let you give the dancer or merge.

@RussGlover thanks again :)

**Comment by etpinard** [comment]
:dancer:

**Pull request merged**